Source/control.cpp

```diff
@@ -1225,12 +1185,9 @@ void FreeControlPan()
 
 bool control_WriteStringToBuffer(BYTE *str)
 {
-	int k;
-	BYTE ichar;
-
-	k = 0;
-	while (*str) {
-		ichar = gbFontTransTbl[*str];
+	int k = 0;
+	while (*str != 0) {

```

Comment:
while (*str != '\0') for checking if the null terminator has been reached?
(Many cases of this, not just this one)